datatables/config.yml

```diff
@@ -0,0 +1,50 @@
+---
+title: Creating a Data Table with CSV
+description: Overview of creating a Data Table from a CSV file
+categories:
+- Architect
+steps:
+- title: Introduction
+  content: |
+    Data tables allow organizations to store data locally and ensure that Architect can access it within an interaction. They can be used to access data sets larger than allowed by a switch statement. 
+    
+    This tutorial demonstrates how to create a data table and import information from a CSV.
+
+    NOTE: Though its contents can be appended to, and modified, after creation, it's not a dedicated database solution. Data tables are not recommended for use where fast and frequent data updating is required.
+
+    For mor information on data tables see the [Resource Center](https://help.mypurecloud.com/articles/work-with-data-tables/)
+
+- title: Authenticate with Genesys Cloud
+  content: |
+    The OAuth Client Credentials are retrieved from environment variables and used to authenticate with Genesys Cloud.
```

Comment:
Do we want to link to any docs or examples of how to authenticate with Genesys Cloud.   I saw later in the code you provide the code example, but this might be a good opportunity to link them to our doco on authenticating.